geospace-lib/src/main/scala/com/socrata/geospace/lib/Utils.scala

```diff
@@ -15,8 +15,8 @@ object Utils extends Logging {
   // Returns (FreeMemoryInMB, FreeMemoryPercentageInt (0-100))
   private def getFree: (Int, Int) = {
     val freeMB = Runtime.getRuntime.freeMemory / MB
-    val totalMB = Runtime.getRuntime.maxMemory / MB
-    val freePct = freeMB * Pct / totalMB
+    val total = Runtime.getRuntime.maxMemory
+    val freePct = freeMB * Pct * MB / total // not converting total now is less expensive
```

Comment:
BTW guys, I've been thinking we should change the overall scheme from free % to just requiring an absolute amount of free memory, which is more reliable.